bats/tests/helpers/vm.bash

```diff
@@ -18,7 +18,7 @@ factory_reset() {
         sudo iptables -L | awk '/^Chain CNI/ {print $2}' | xargs -l sudo iptables -X
```

Comment:
Please, quote $2 in `{print "$2"}`

Reply:
Nope, this is not shell/bash, it is `awk` code. And if you quote it, it will print a literal `$2` instead of the value of it, which is not what we want.